### ISSUE TITLE
./waf failure in building ChronoChat Application (Cannot find <ndn-cxx/common.hpp>)

### ISSUE THREAD

**Issue by shruthikmusukula** [named-data/ChronoChat#5]
Here is the error I am facing when building the ChronoChat application. For some reason when looking at build/config.log, no errors are detected. How can it detect ndn-cxx.common.hpp? I faced this error when building the application on macOS and now again when attempted in an Ubuntu 18.04 Virtual Machine.

```
Waf: Entering directory `/home/shruthikm/NamedData/ChronoChat/build'
[14/52] Compiling src/tree-layout.cpp
[17/53] Compiling src/validator-invitation.cpp
In file included from ../src/trust-tree-node.hpp:14:0,
                 from ../src/tree-layout.hpp:16,
                 from ../src/tree-layout.cpp:13:
../src/common.hpp:31:10: fatal error: ndn-cxx/common.hpp: No such file or directory
 #include <ndn-cxx/common.hpp>
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.

In file included from ../src/validator-invitation.hpp:14:0,
                 from ../src/validator-invitation.cpp:11:
../src/common.hpp:31:10: fatal error: ndn-cxx/common.hpp: No such file or directory
 #include <ndn-cxx/common.hpp>
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.

Waf: Leaving directory `/home/shruthikm/NamedData/ChronoChat/build'
Build failed
 -> task in 'ChronoChat' failed (exit status 1): 
	{task 140397622452944: qxx tree-layout.cpp -> tree-layout.cpp.1.o}
['/usr/bin/g++', '-std=c++0x', '-std=c++11', '-Wall', '-Wno-nested-anon-types', '-O2', '-g', '-pthread', '-pthread', '-I/home/shruthikm/NamedData/ChronoChat/build/src', '-I/home/shruthikm/NamedData/ChronoChat/src', '-I/home/shruthikm/NamedData/ChronoChat/build', '-I/home/shruthikm/NamedData/ChronoChat', '-I/usr/include/qt4', '-I/usr/include/qt4/QtCore', '-I/usr/include/qt4/QtGui', '-I/usr/include/qt4/QtSql', '-I/usr/local/include', '-I/usr/include', '-I/usr/local/include/ChronoSync', '-DWAF=1', '-DHAVE_QTCORE=1', '-DQT_SHARED', '-DHAVE_QTGUI=1', '-DQT_SHARED', '-DHAVE_QTSQL=1', '-DQT_SHARED', '-DHAVE_NDN_CXX=1', '-DBOOST_STACKTRACE_DYN_LINK', '-DBOOST_LOG_DYN_LINK', '-DHAVE_SYNC=1', '../src/tree-layout.cpp', '-c', '-o', '/home/shruthikm/NamedData/ChronoChat/build/src/tree-layout.cpp.1.o']
 -> task in 'ChronoChat' failed (exit status 1): 
	{task 140397622453200: qxx validator-invitation.cpp -> validator-invitation.cpp.1.o}
['/usr/bin/g++', '-std=c++0x', '-std=c++11', '-Wall', '-Wno-nested-anon-types', '-O2', '-g', '-pthread', '-pthread', '-I/home/shruthikm/NamedData/ChronoChat/build/src', '-I/home/shruthikm/NamedData/ChronoChat/src', '-I/home/shruthikm/NamedData/ChronoChat/build', '-I/home/shruthikm/NamedData/ChronoChat', '-I/usr/include/qt4', '-I/usr/include/qt4/QtCore', '-I/usr/include/qt4/QtGui', '-I/usr/include/qt4/QtSql', '-I/usr/local/include', '-I/usr/include', '-I/usr/local/include/ChronoSync', '-DWAF=1', '-DHAVE_QTCORE=1', '-DQT_SHARED', '-DHAVE_QTGUI=1', '-DQT_SHARED', '-DHAVE_QTSQL=1', '-DQT_SHARED', '-DHAVE_NDN_CXX=1', '-DBOOST_STACKTRACE_DYN_LINK', '-DBOOST_LOG_DYN_LINK', '-DHAVE_SYNC=1', '../src/validator-invitation.cpp', '-c', '-o', '/home/shruthikm/NamedData/ChronoChat/build/src/validator-invitation.cpp.1.o']
```

**Comment by BITXUJI** [comment]
The same error!  Have you fixed it? I can not find this file in ndn-cxx source code too.

**Comment by Someoneece** [comment]
Please some one solve this problem and help us to build the app

**Comment by Pesa** [comment]
Should be fixed as of commit 3d85090003bd3918b0c0263636847c8840e4cfe1
